crates/node-maintainer/src/maintainer.rs

```diff
@@ -187,7 +188,16 @@ impl NodeMaintainer {
     }
 
     pub async fn extract_to(&self, path: impl AsRef<Path>) -> Result<(), NodeMaintainerError> {
-        async fn inner(me: &NodeMaintainer, path: &Path) -> Result<(), NodeMaintainerError> {
+        let pb_style = ProgressStyle::default_bar()
+            .template("💾 {bar:40} [{pos}/{len}] {wide_msg}")
+            .unwrap();
+        let pb = ProgressBar::new(self.graph.inner.edge_count() as u64).with_style(pb_style);
```

Comment:
2 things:

1. Anywhere that we use `indicatif` should only do things on non-wasm32, and should be conditionalized appropriately.
2. There should be a configuration, through `NodeMaintainerOptions`, that configures whether or not progress bars should be shown. This should default to false (because the default assumption is for node-maintainer to be used as a plain library without unwanted output)

Reply:
Hopefully achieved what you meant here:
https://github.com/orogene/orogene/pull/145/commits/ed6711d4fdc9967a17d1898487d1218c691a7e55

If it's way too ugly please let me know!